testing/travis-fork-check.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
+import os
+import subprocess
+import sys
+
+def check_travis_repo_slug():
+    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
+        logging.warning("Decryption disabled. Setting MANTL_CI_FORK_CHECK to 1")
+        print("1")
+    else:
+        logging.info("Decrypting OS ssh key")
```

Comment:
Yes. The plan was to have the Travis instance log into a secure SSH jump box to run OS tests. Because we need to have the same key across different instances, I added the key as an encrypted file to the repo
> On May 11, 2016, at 12:44, Brian Hicks <notifications@github.com> wrote:
> 
> In testing/travis-fork-check.py <https://github.com/CiscoCloud/mantl/pull/1415#discussion_r62893046>:
> 
> > @@ -0,0 +1,30 @@
> > +#!/usr/bin/env python
> > +from __future__ import print_function
> > +import logging
> > +import os
> > +import subprocess
> > +import sys
> > +
> > +def check_travis_repo_slug():
> > +    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
> > +        logging.warning("Decryption disabled. Setting MANTL_CI_FORK_CHECK to 1")
> > +        print("1")
> > +    else:
> > +        logging.info("Decrypting OS ssh key")
> Is this only for OpenStack?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/CiscoCloud/mantl/pull/1415/files/dce2ccb628c69104764c0cad923e016abacadee1#r62893046>